userspace/engine/falco_engine_version.h

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
 // The version of this Falco engine
 #define FALCO_ENGINE_VERSION_MAJOR 0
-#define FALCO_ENGINE_VERSION_MINOR 39
+#define FALCO_ENGINE_VERSION_MINOR 40
```

Comment:
I think we can avoid bumping the engine in this case. This is effectively a follow-up to the same change we made few days ago.